.codeclimate.yml

```diff
@@ -29,23 +32,11 @@ checks:
       threshold: # language-specific defaults. an override will affect all languages.
     exclude_patterns:
       - '*/engine.rb'
-      - "**/swagger/**.rb"
-  identical-code:
-    config:
-      threshold: # language-specific defaults. an override will affect all languages.
+      - '**/swagger/**.rb'
```

Comment:
Github diff might be playing tricks on my eyes...

![image](https://user-images.githubusercontent.com/3077884/81949343-c3447080-95d0-11ea-9bd9-d86b91b92ebf.png)

Does this `- '**/swagger/**.rb'` need to move up to line 15 where it looks like you moved the `identical-code:` element (presumably alphabetized).

Reply:
I think '**/swagger/**.rb' is in the correct place/nested underneath exclude_patterns. The only difference between the swagger portion of the diff is the single vs double quotes. I think that answers the question? Heres' the original ordering and nesting: 
<img width="900" alt="Screen Shot 2020-05-14 at 10 58 54 AM" src="https://user-images.githubusercontent.com/5834818/81950469-f804f780-95d1-11ea-93bb-ba0ce8841a45.png">